content/documentation/langium/content.md

```diff
@@ -1,13 +1,23 @@
 +++
 fragment = "content"
 title = "Langium Integration"
-weight = 150
+weight = 160
 [sidebar]
   sticky = true
 +++
 
-TODO say what langium is and link to it
+To extend the Model Hub with your specific language, you need to provide a `ModelServiceContribution`.
+A model service contribution can provide a dedicated Model Service API and extend the persistence and validation capabilities of the Model Hub.
```

Comment:
I think this is the main part of the integration between model hub and Langium, so we may want to extend this paragraph a bit.

Model Service is specific to model hub ("apply any changes you like to the model"), but persistence and validation are typically delegated to the Langium language server. So I think we should have an example of that (maybe a simple copy of our current CoffeeModelServiceContribution example? But you mentioned it might change, so I don't know if it's "good enough" to be used as an example, or if it's too outdated).

Either way, in this section, I would focus on these 2 key aspects:

- persistence
- validation

and then just link to the Model Hub documentation for the Model Service itself.

Reply:
Thank you, I added additional content for those two areas!